simtools/utils/general.py

```diff
@@ -124,9 +124,17 @@ def validate_config_data(config_data, parameters, ignore_unidentified=False):
         if par_name in out_data:
             continue
         if "default" in par_info.keys() and par_info["default"] is not None:
-            default_value = par_info["default"]
+            if isinstance(par_info["default"], dict):
+                default_value = par_info["default"]["value"]
+                default_value = (
+                    default_value * u.Unit(par_info["default"]["unit"])
```

Comment:
I have tried to merge `_convert_to_quantities` with those in general.py without success. Spent a lot of time and deleted the non-working code simply because configuration files for simtel and corsika are quite fine tuned.

I know this is not ideal, but I don't see a way to get a very clean code when the config / input files to be written require this clear and fixed format.

Reply:
Ok, thanks for trying it out. Given that it is not straightforward, I am ok with this as well.